dll/win32/uxtheme/png.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     PNG image support in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <Splitwirez@gmail.com>
+ */
+
+#include "uxthemep.h"
+
+#ifdef MSSTYLES_ACCEPT_PNG_IMAGES
+#include <ole2.h>
+#include <objidl.h>
+#include <gdiplus.h>
+#include <gdipluscolor.h>
+
+BOOL MSSTYLES_TryLoadPng(HINSTANCE hTheme, LPCWSTR szFile, LPCWSTR type, OUT HBITMAP *pngBitmap)
```

Comment:
Since you kind of want to use annotations, here's something better:
```suggestion
BOOL
MSSTYLES_TryLoadPng(
    _In_ HINSTANCE hTheme,
    _In_ LPCWSTR szFile,
    _In_ LPCWSTR type,
    _Out_ HBITMAP* pngBitmap)
```

Reply:
Also here this function is compiled as C++, so I'm not sure that this block in the header is very correct... :
```c
#ifdef __cplusplus
extern "C" {
#endif
BOOL MSSTYLES_TryLoadPng(HINSTANCE hTheme, LPCWSTR szFile, LPCWSTR type, OUT HBITMAP *pngBitmap);
#ifdef __cplusplus
```

Reply:
I don't see any other function in here that's using SAL2?
Should he really be annotating a single function in the entire DLL?
Up to you though.